global_planner:
  allow_unknown: false
  default_tolerance: 0.0
  visualize_potential: true
  use_quadratic: true
  use_grid_path: false
  old_navfn_behavior: false

  plugins:
    - { name: static_layer, type: "costmap_2d::StaticLayer" }
    - { name: obstacle_layer, type: "costmap_2d::ObstacleLayer" }
    - { name: inflation_layer, type: "costmap_2d::InflationLayer" }

footprint: [[0.145, 0.25], [0.145, -0.25], [-0.455, -0.25], [-0.455, 0.25]]
footprint_padding: 0.00

costmap:
  global_frame: "map"
  robot_base_frame: "base_link"
  update_frequency: 3.0
  publish_frequency: 3.0
  rolling_window: false
  transform_tolerance: 5.0

  obstacle_layer:
    enabled: true
    combination_method: 1
    track_unknown_space: true
    obstacle_range: 2.5
    raytrace_range: 3
    observation_sources: laser_scan_sensor
    laser_scan_sensor:
      {
        sensor_frame: lidar_link,
        data_type: LaserScan,
        topic: openmover/hardware/lidar/scan,
        marking: true,
        clearing: true,
      }

  plugins:
    - { name: static_layer, type: "costmap_2d::StaticLayer" }
    - { name: obstacle_layer, type: "costmap_2d::ObstacleLayer" }
    - { name: inflation_layer, type: "costmap_2d::InflationLayer" }

  static_layer:
    enabled: true
    map_topic: "openmover/mapping/map"

  inflation_layer:
    enabled: true
    cost_scaling_factor: 3.0
    inflation_radius: 0.0